releases/mainnet/1.2.5/index.html

```diff
@@ -0,0 +1,24 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.2.5</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.2.5</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.2.5/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmWGjaY1pacjp9T8JAQ9VbrJWBsgsNzgvwduDccr1gMjUq">Contract addresses (IPFS hosted)</a></li>
```

Comment:
This ipfs JSON doesn't match releases/mainnet/1.2.5/contracts.json  file.

Reply:
I'll take care of this.

Reply:
added